src/api/java/com/minecolonies/api/colony/IColonyManager.java

```diff
@@ -432,4 +432,10 @@ static IColonyManager getInstance()
      * Reset all colony views on login to new world.
      */
     void resetColonyViews();
+
+    /**
+     * Open the new reactivation window.
+     * @param pos the pos to open it at.
+     */
+    void openReactivationWindow(final BlockPos pos);
```

Comment:
why is this here in the colony manager?

Reply:
Because of api/main seperation, that is the most accessible place from AbstractBlockHut I could find.